CONTRIBUTING.md

```diff
@@ -60,7 +62,9 @@ Once you've worked out details with the maintainers, follow these general steps:
 Also add the command name to `envlist`.
 1. Write your tutorial!
     * For notebook-based tutorials, write your tutorial either as a Python script (e.g. `my_tutorial_dir/my_tutorial.py`) in [Jupytext percent format](https://gist.github.com/mwouts/91f3e1262871cdaa6d35394cd14f9bdc) or a Jupyter notebook.
-        * Run `tox -e my_tutorial_dir -- sync` to generate a notebook version from the Python script version. Run this command to update when changes are made to the tutorial script.
+        * Run `tox -e my_tutorial_dir -- sync` to generate a notebook version from the Python script version
+          (or if you have run all cells, you can select `File` &rarr; `Jupytext` &rarr; `Pair Notebook with Percept Script`).
+          Do this to update the notebook whenever changes are made to the tutorial script.
```

Comment:
We need to pair every time we make changes?

Reply:
Updated instructions to unpair after pairing to prevent jupyter from spuriously saving the notebook.

Reply:
just realized that this requires a jupytext install. and in fact, need to launch the notebook from a sufficient virtualenv. if you rebase, there are some dev instructions. can you add instructions for activating the relevant virtualenv for folks working in notebooks in that section

Reply:
Added. jupytext is in the requirements, so it'll get installed in the pip install -r requirements.txt.